### DISCUSSION TITLE
PrimeNG SASS Theme to Design tokens

### DISCUSSION THREAD

**Discussion by djay1977** [primefaces/community#3260] [PrimeNG]
Hello,
Do you plan a tool to migrate the sass theme from primeng (https://github.com/primefaces/primeng-sass-theme) to the new theming system (Design Tokens) ?

Regards,
Jerry

**Comment by qwadrox** [comment]
@djay1977 
I don't think it's technically feasible. Lots of things changed and the previous system cannot be mapped or converted to the new. 
They explicitly mentioned: 
> If you had a custom theme for v17, the theme needs to be recreated using the new APIs

https://primeng.org/guides/migration#breaking

**Comment by djay1977** [comment]
Yes, I know the migration guide and I have already done it on some projects in full 19.
However, on some of my projects that use the PrimeNG sass api theme, I think it's possible to create a utility to migrate the variables from the SASS API to the Design Tokens and I wanted to know if Prime had planned to do it?
